docs/samples/triton/bert/bert.yaml

```diff
@@ -25,6 +25,7 @@ spec:
               value: "gs://kfserving-samples/models/triton/bert-transformer"
     predictor:
       triton:
+        runtimeVersion: 20.03-py3
```

Comment:
This version of triton doesn't use V2 protocol. You should update this to recent version (20.10).

Reply:
this is the existing KFServing v1alpha2 example which still uses the old triton version, so just make the version more explicit.